recipes/ldas-tools-ldasgen/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ldas-tools-ldasgen" %}
+{% set version = "2.6.3" %}
+{% set sha256 = "e999263c2341a3935b5030c83304ce423da0aee2b3f719a86a172e5dad5b74ba" %}
+{% set ldas_tools_al_version = "2.6.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+  host:
+    - flex
+    - boost-cpp
+    - openssl
+    - zlib
+    - ldas-tools-al >={{ ldas_tools_al_version }}
+  run:
+    - ldas-tools-al >={{ ldas_tools_al_version }}
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/genericAPI  # [not win]
+    - test -f ${PREFIX}/lib/libStat${SHLIB_EXT}  # [not win]
+    - test -f ${PREFIX}/lib/libldasgen${SHLIB_EXT}  # [not win]
+
+about:
+  home: https://wiki.ligo.org/Computing/DASWG/LDASTools
+  dev_url: https://git.ligo.org/ldastools/LDAS_Tools
+  license: GPLv2
+  license_family: GPL
+  license_file: COPYING
+  summary: Filters library used by ldas-tools
+  description: |
+    This provides the runtime libraries for the filters library.
+
+extra:
+  recipe-maintainers:
+    - duncanmmacleod
+    - emaros
```

Comment:
@emaros should comment here that they are ok maintaining these recipes.

Reply:
I am ok being listed as a recipe-maintainer